src/typeDefs/types.ts

```diff
@@ -294,6 +311,26 @@ export const types = gql`
     aggregate: AggregatePost!
   }
 
+  type Tag {
+    _id: ID!
+    name: String!
+    organization: Organization!
+    parentTag: Tag
+    childTags: [Tag]
+    usersAssignedTo: [User]
+    eventsAssignedTo: [Event]
+  }
+
+  type TagsAssignedWithConnection {
```

Comment:
You don't need a different name for each connection related to same object. Just name it `TagsConnection` and use it wherever you're returning a big list of tags.